src/ActivitiesSDKAdapter.js

```diff
@@ -84,4 +92,35 @@ export default class ActivitiesSDKAdapter extends ActivitiesAdapter {
 
     return this.activityObservables[ID];
   }
+
+  /**
+   * Creates an activity and returns an observable to the new activity data
+   *
+   * @param {object} activity  Adapter activity object
+   * @returns {Observable.<Activity>} Observable stream that emits data of the newly activity
+   */
+  postActivity(activity) {
+    const cardActivity = activity.card && [
```

Comment:
you can use just `card`. `cardActivity` implies that this variable holds an activity object (of type card - compare with redActivity)